static/src/styles/tasklist.css

```diff
@@ -171,6 +171,11 @@
   color: #0b0c0c;
 }
 
+.signout {
+  padding-left: 0px;
+  margin-top: 0px;
```

Comment:
You would think it all would have been padding but no. The margin top was causing a gap on this